club/models.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Member(AbstractUser):
+    email = models.EmailField(unique=True)
+    type = models.CharField(max_length=20)
+    address = models.CharField(max_length=100)
+    home_phone = models.CharField(max_length=20)
+    mobile = models.CharField(max_length=20)
+    date_added = models.DateTimeField(auto_now_add=True)
+    
+    def __str__(self):
+        return f'{self.first_name} {self.last_name}'
+    
+    
+class Booking(models.Model):
+    IS_RECURRING_CHOICES = [(False, 'No'), 
+                            (True, 'Yes')]
+    time = models.DateTimeField()
+    start_date = models.DateTimeField()    
+    end_date = models.DateTimeField()
+    date_added = models.DateTimeField(auto_now_add=True)
+    recurring = models.BooleanField(default=False, 
+                                    choices=IS_RECURRING_CHOICES)
+    member = models.ForeignKey(Member, 
+                               on_delete=models.SET_NULL, 
+                               related_name='booking')
+
+    class Meta:
+        ordering = ['-time']
+
+    def __str__(self):
+        return f'{self.time} - {self.member}'
+    
+
+class Guests(models.Model):
+    players = models.ManyToManyField(Booking)
+    date_added = models.DateTimeField(auto_now_add=True)
+    member_id = models.CharField(max_length=10)
```

Comment:
Is member_id needed ( the primary key from the member table)? Would need to be able to know the other members that are included in each booking (to know when they play)

Reply:
If member_id is a foreign key to the Member table I would expect that type of column, not CharField?

Reply:
I don't see this table in the diagram? https://github.com/theonlysam/bookthecourt/issues/2#issuecomment-1663265360 

What is the idea here?

Reply:
I thought we would go with a many-to-many field called "guests" on the Booking model for this? This would assume though that all people on a booking are members / users (so on the system).